README.md

```diff
@@ -52,12 +52,13 @@ Refer [hello_world.ipynb](https://github.com/mozilla/overscripted/blob/master/an
 - The dataset is very large. Even the subsets of the dataset are unlikely to fit into memory. Working with this dataset will typically require using Dask (http://dask.pydata.org/), Spark (http://spark.apache.org/) or similar tools to enable parallelized / out-of-core / distributed processing.
 
 ### Glossary
-
+- [Fingerprinting](https://en.wikipedia.org/wiki/Device_fingerprint#Limitations) is the process of creating a unique identifier based off of some characteristics of your hardware, operating  system and browser. It can be used to fully or partially identify individual users or devices even when persistent cookies (and also zombie cookies) can't be read or stored in the browser, the client IP address is hidden, and even if one switches to another browser on the same device.
```

Comment:
The last part of this sentence is almost certainly not true. A different browser should give a very different fingerprint. Cross-device/platform re-identification is something that trackers do, but it's a different mechanism.

The link you've included is directly to the limitations section which doesn't seem appropriate for the main link for this section. In addition, the limitations section that you've link confirms what I wrote above and contradicts your statement.

Reply:
@birdsarah Thanks for the review and clarifying that a different browser should give a different fingerprint.
Removed the last the sentence and Updated the link as well.
Ready for Review.